packages/paste-core/components/badge/src/StatusBadge.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
```

Comment:
Normally package exports should be prefixed with the package name, so something like `BadgeStatus`. I like `StatusBadge` more, but we also have a similar thing in Menu in this PR: `MenuBadgeStatus`.

So either we rename this `BadgeStatus` 🤢  or we renamed `MenuBadgeStatus` to `MenuStatusBadge`

Reply:
Yeah, it's awkward. I too really dislike `BadgeStatus`. The problem I faced with Menu was that there is actually a `MenuBadge`, and I was really struggling with:

- `MenuButton`
- `MenuBadge`
- `MenuStatusBadge`

Gunna throw a straw poll up in Slack to find our least worst preference